pom.xml

```diff
@@ -339,7 +339,7 @@
         <version.coveralls.plugin>4.3.0</version.coveralls.plugin>
         <version.com.buschmais.jqassistant.plugin>1.8.0</version.com.buschmais.jqassistant.plugin>
         <version.docker.maven.plugin>0.34.1</version.docker.maven.plugin>
-        <version.moditect.plugin>1.0.0.Beta2</version.moditect.plugin>
+        <version.moditect.plugin>1.0.0-SNAPSHOT</version.moditect.plugin>
```

Comment:
Please make sure to change this to a published version before we merge the PR.

Reply:
This request will always be a draft as long as a new Beta of ModiTect is not released.